content/docs/Query.md

```diff
@@ -9,26 +9,90 @@ identifier = 'queries'
 
 +++
 
-Queries supported by bleve:
+Queries supported by bleve. Note the example code on this page assumes you have created a bleve index with the sample data from the [beer-search](https://github.com/blevesearch/beer-search) application.
 
 ### Term
 
 A term query is the simplest possible query.  It performs an exact match in the index for the provided term.
 
 Most of the time users should use a Match Query instead.
 
+```go
+package main
+
+func main() {
+    index, err := bleve.Open("beer-search.bleve")
+    if err != nil {
+        log.Fatal(err)
+    }
+
+    q := bleve.NewTermQuery("golden")
+    req := bleve.NewSearchRequest(q)
+    req.Highlight = bleve.NewHighlightWithStyle("html")
```

Comment:
I'm thinking that to keep these examples short and focused on how to use the query type being discussed, we should remove the lines which set `req.Highlight`.